packages/react/src/components/Search/Search-test.js

```diff
@@ -97,13 +96,6 @@ describe('Search', () => {
         });
       });
 
-      describe('icons', () => {
```

Comment:
Why did you remove the tests here?

Reply:
I removed the tests related to the Icon component because as we discussed last time, it will be deprecated and hence not worth have any test related to it. Let me know if you have any other thoughts :)